omnigibson/simulator.py

```diff
@@ -1023,7 +1038,9 @@ def play(self):
                 if gm.ENABLE_FLATCACHE:
                     channels.append("omni.physx.plugin")
                 with suppress_omni_log(channels=channels):
-                    super().play()
+                    # Playing takes some steps, so we temporarily change the dt to 0 to avoid moving things.
+                    with self.slowed():
```

Comment:
FYI when i tried slowing and unslowing the sim back years ago (maybe during the 4.0.0 or even before??), the resulting unslowed-sim could be unstable. not sure if that's still an issue?